src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml

```diff
@@ -119,6 +119,11 @@ tests:
           "Step 4: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID and Breadcrumb field
           set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```

Comment:
In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

Reply:
Added